### PR TITLE
MIG-766: proxy config

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MIG-766

Added proxy config module to installation assemblies. 
Removed Stunnel proxy configuration module and added Stunnel to new proxy config module.

4.6+

Preview: https://deploy-preview-35065--osdocs.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/installing-3-4.html#migration-configuring-proxies_installing-3-4

QE approved. Ready for peer review.

### CODE REVIEW

**Comment by XinRedhat** [comment]
LGTM

**Review by sergiordlr — APPROVED** [review]
Looks good to me. Thank you very much!

**Review by EricPonvelle — CHANGES_REQUESTED** [review]
Small suggestion

File: modules/migration-configuring-proxies.adoc [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+//
+// * migrating_from_ocp_3_to_4/installing-3-4.adoc
+// * migrating_from_ocp_3_to_4/installing-restricted-3-4.adoc
+// * migration_toolkit_for_containers/installing-mtc.adoc
+// * migration_toolkit_for_containers/installing-mtc-restricted.adoc
+
+[id="migration-configuring-proxies_{context}"]
+= Configuring proxies
+
+For {product-title} 3.x and 4.1, you must configure proxies in the `MigrationController` custom resource (CR) manifest after you install the {mtc-short} Operator because these versions do not support a cluster-wide `proxy` object.
```
[the remark this answers is not in the document]
Comment (apinnick) [reply modules/migration-configuring-proxies.adoc:11]: No, that is the correct name of the Operator. I am referring to the MTC Operator, not the product.

**Pull request merged**